src/main/steps/applicant1Sequence.test.ts

```diff
@@ -1,6 +1,6 @@
 import { ApplicationType } from '../app/case/definition';
 
-import { Step, applicant1Sequence, isCountryUk } from './applicant1Sequence';
+import { Step, applicant1PreSubmissionSequence, isCountryUk } from './applicant1Sequence';
```

Comment:
if it's not too much trouble I would use the names:

```
applicant1ApplicationSequence
applicant2....
applicant1ConditionalOrderSequence
applicant1FinalOrderSequence
```

I'm not sure if we have a sequence for CO and FO but it would logically line up with the application phases if we did.

Reply:
looking at the file below it seems like we just have pre and post submission so this comment might not be so relevant but I'll leave it up to you